### PR TITLE
Google Cloud Diagnostics ASP.NET Core Error Reporting Integration Tests

### PR DESCRIPTION
**Pull request by iantalarico**

### CODE REVIEW

File: apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/ErrorReporting/ErrorReportingTest.cs [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+    public class ErrorReportingTest
+    {
+        /// <summary>Total time to spend sleeping when looking for error events.</summary>
+        private readonly TimeSpan _timeout = TimeSpan.FromSeconds(10);
+
+        /// <summary>Time to sleep between checks for error events.</summary>
+        private readonly TimeSpan _sleepInterval = TimeSpan.FromSeconds(2);
+
+        /// <summary>A query time range of one hour.</summary>
+        private readonly QueryTimeRange s_oneHour = new QueryTimeRange { Period = Period._1Hour };
```
Comment (evildour) [inline apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/ErrorReporting/ErrorReportingTest.cs]: Should these 3 fields be static?

File: apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/ErrorReporting/ErrorReportingTest.cs [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+                {
+                    ListEventsRequest request = new ListEventsRequest
+                    {
+                        ProjectName = _projectName.ToString(),
+                        GroupId = group.Group.GroupId,
+                        TimeRange = s_oneHour
+                    };
+
+                    var events = _client.ListEvents(request);
+                   errorEvents.AddRange(events.Where(e => e.Message.Contains(testId)));
```
Comment (evildour) [inline apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/ErrorReporting/ErrorReportingTest.cs]: Nit: indent is a little off here.

File: apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/ErrorReporting/ErrorReportingTest.cs (hunk 1 above)
Comment (iantalarico) [inline apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/ErrorReporting/ErrorReportingTest.cs]: Yes, done.

File: apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/ErrorReporting/ErrorReportingTest.cs (hunk 2 above)
Comment (iantalarico) [inline apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/ErrorReporting/ErrorReportingTest.cs]: Fixed.

**Pull request merged**